docker/run.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+rm -rf build
+mkdir build
+
+docker run --rm -t -v $PWD/build:/opt/siege -v $PWD/compile.sh:/opt/compile.sh ubuntu bash /opt/compile.sh
```

Comment:
Personally, I like using the virtual package option of apk to install dependencies, compile, then uninstall dependencies - all in one layer. That way a single `docker build` results in a light, functional image - without the need for intermediary containers as in line 6.